lib/src/main/java/com/basistheory/android/view/TextElement.kt

```diff
@@ -110,7 +125,21 @@ class TextElement : FrameLayout {
         subscribeToInputEvents()
     }
 
+    private fun addMask(mask: List<Any>?) {
+        if (!mask.isNullOrEmpty()) {
+            maskWatcher = MaskWatcher(mask)
+
+            if (maskWatcher != null) {
+                input.inputType = TYPE_CLASS_NUMBER // temporarily hardcoding when there's a mask
+                input.addTextChangedListener(maskWatcher);
+            }
+        }
```

Comment:
Looks like there's no way to set a mask and remove it. We short circuit here if it's set to null without removing the old watcher.